plugins/woocommerce/src/Internal/Admin/Settings.php

```diff
@@ -288,6 +288,19 @@ public function add_settings( $settings ) {
 			'default'     => 'period=month&compare=previous_year',
 			'type'        => 'text',
 		);
+		$settings[] = array(
+			'id'          => 'woocommerce_date_type',
+			'option_key'  => 'woocommerce_date_type',
+			'label'       => __( 'Date Type', 'woocommerce' ),
+			'description' => __( 'Database date field considered for Revenue and Orders reports', 'woocommerce' ),
+			'default'     => 'date_paid',
```

Comment:
```suggestion
			'default'     => 'date_created',
```

I noticed that the analytics -> settings -> date type setting is defaulting to `date_paid`, likely because of this line? I think we want this to remain on `date_created`, right?

![image](https://user-images.githubusercontent.com/444632/219130801-ddb3aa05-7d5c-4fe8-83bb-0b2260a2ec58.png)